utils/proxy-tools.ts

```diff
@@ -0,0 +1,32 @@
+// SPDX-FileCopyrightText: 2024 IEXEC BLOCKCHAIN TECH <contact@iex.ec>
+// SPDX-License-Identifier: Apache-2.0
+
+import { getFunctionSignatures } from '../migrations/utils/getFunctionSignatures';
+import { ERC1538Update } from '../typechain';
+import { getBaseNameFromContractFactory } from '../utils/deploy-tools';
+
+/**
+ * Link a contract to an ERC1538 proxy.
+ * @param proxy contract to ERC1538 proxy.
+ * @param contractAddress The contract address to link to the proxy.
+ * @param contractFactory The contract factory to link to the proxy.
+ */
+export async function linkContractToProxy(
```

Comment:
Where do you use it?

Reply:
In `deploy/0_deploy.ts`.
I like the fact that it is separated in a utility file (since it's already done).

Reply:
I like it too, I just missed it thanks :)